### ISSUE TITLE
🐛 BUG: Wrangler Init fails with: ERROR Raw mode is not supported using the Git for windows terminal

### ISSUE THREAD

**Issue by socketopp** [cloudflare/wrangler2#1992]
### What version of `Wrangler` are you using?

2.1.10

### What operating system are you using?

Windows

### Describe the Bug

This error occured today when I tried to install wrangler. 

Currently using node v16.17.1
Using [Git for windows terminal](https://gitforwindows.org/) v.2.28.0

Note: Everything works fine in Windows Command prompt

```
$ npx wrangler init failedge-worker
 Ôøà´©Å wrangler 2.1.10
--------------------
Using npm as package manager.
Ô£¿ Created failedge-worker\wrangler.toml
Error: Raw mode is not supported on the current process.stdin, which Ink uses as input stream by default.
Read about how to prevent this error on https://github.com/vadimdemedes/ink/#israwmodesupported
    at App.handleSetRawMode (~\npm\node_modules\wrangler\wrangler-dist\cli.js:71229:21)
    at ~\npm\node_modules\wrangler\wrangler-dist\cli.js:71890:9
    at jh (~\npm\node_modules\wrangler\wrangler-dist\cli.js:20431:26)
    at exports2.unstable_runWithPriority (~\npm\node_modules\wrangler\wrangler-dist\cli.js:16583:16)
    at Pc (~\npm\node_modules\wrangler\wrangler-dist\cli.js:17198:16)
    at Qg (~\npm\node_modules\wrangler\wrangler-dist\cli.js:20383:18)
    at ~\npm\node_modules\wrangler\wrangler-dist\cli.js:20373:13
    at V (~\npm\node_modules\wrangler\wrangler-dist\cli.js:16509:22)
    at Timeout.w [as _onTimeout] (~\npm\node_modules\wrangler\wrangler-dist\cli.js:16350:13)
    at listOnTimeout (node:internal/timers:559:17)

  ERROR Raw mode is not supported on the current process.stdin, which Ink uses
       as input stream by default.
       Read about how to prevent this error on
       https://github.com/vadimdemedes/ink/#israwmodesupported

 - Read about how to prevent this error on
   https://github.com/vadimdemedes/ink/#israwmodesupported
 -App.handleSetRaw ~/AppData/Roaming/npm/node_modules/wrangler/wr
  ode             angler-dist/cli.js:71229:21)
 - ~/AppData/Roaming/npm/node_modules/wrangler/wrangler-dist/cli.
  js:71890:9)
 -jh ~/AppData/Roaming/npm/node_modules/wrangler/wrangler-dist/cl
    i.js:20431:26)
 -exports2.unstable_runWi ~/AppData/Roaming/npm/node_modules/wran
  hPriority              gler/wrangler-dist/cli.js:16583:16)
 -Pc ~/AppData/Roaming/npm/node_modules/wrangler/wrangler-dist/cl
    i.js:17198:16)
 -Qg ~/AppData/Roaming/npm/node_modules/wrangler/wrangler-dist/cl
    i.js:20383:18)
 - ~/AppData/Roaming/npm/node_modules/wrangler/wrangler-dist/cli.
  js:20373:13)
 -V ~/AppData/Roaming/npm/node_modules/wrangler/wrangler-dist/cli
   .js:16509:22)
 -Timeout. ~/AppData/Roaming/npm/node_modules/wrangler/wrangler-d
          ist/cli.js:16350:13)
 - listOnTimeout (node:internal/timers:559:17)
```

**Comment by rozenmd** [comment]
Suggesting this as a quick win:

We should check if [`isRawModeSupported`](https://github.com/vadimdemedes/ink/#israwmodesupported) when outside of CI. If not, they're running an unsupported terminal, and we should let them know to use supported terminals like Command Prompt etc

**Comment by rozenmd** [comment]
Reopened this as it's still an issue in 2.1.13

**Comment by rozenmd** [comment]
Apparently it happens in linux too?

> fwiw I'm in a standard bash terminal in linux

**Comment by caass** [comment]
😭

**Comment by rozenmd** [comment]
Actually @caass - @penalosa's PR to move to `prompts` may inadvertently fix this!

**Comment by rozenmd** [comment]
https://github.com/cloudflare/wrangler2/pull/2283

**Comment by hanzlahabib** [comment]
I see this issue in gitbash terminal

**Comment by rozenmd** [comment]
@socketopp  / @AonghasMackay / @hanzlahabib - do you still get this issue as of wrangler 2.7.1?

**Comment by rozenmd** [comment]
Closing this as fixed in 2.7.1
